src/persist/src/indexed/runtime.rs

```diff
@@ -752,13 +746,50 @@ impl<K: Codec, V: Codec> Iterator for DecodedSnapshotIter<K, V> {
     }
 }
 
+#[derive(Debug, Clone)]
+enum CmdReadSender {
+    Full(crossbeam_channel::Sender<RuntimeCmd>),
```

Comment:
I'm skimming so I may have missed something obvious but why is this variant called Full?

Reply:
ha! this was from you, though I may have misremembered the term you used. the opposite of Read isn't really Write here, it's Read+Write. in most places we can get away with Foo and FooRead variants (the only real exception previous to this is open_exclusive vs open_read, which work because exclusive has its own meaning at the level of the storage trait). definitely open to suggestions

Reply:
ok it makes a lot more sense once the second variant is added in :D